### PR TITLE
Update "Who is Apollo"

### PR DESCRIPTION
**Pull request by Meschreiber**
Updates "Who is Apollo" text and links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ Learn how to use Apollo Client with self-paced hands-on training on Odyssey, Apo
...
+- [GraphOS](https://www.apollographql.com/graphos) - The platform for building, managing, and scaling a supergraph: a unified network of your organization's microservices and their data sources—all composed into a single distributed API.
+- [Apollo Federation](https://www.apollographql.com/federation) – The industry-standard open architecture for building a distributed graph. Use Apollo’s gateway to compose a unified graph from multiple subgraphs, determine a query plan, and route requests across your services.
+- [Apollo Client](https://github.com/apollographql/apollo-client) – The most popular GraphQL client for the web. Apollo also builds and maintains [Apollo iOS](https://github.com/apollographql/apollo-ios) and [Apollo Kotlin](https://github.com/apollographql/apollo-kotlin).
 - [Apollo Server](https://www.apollographql.com/docs/apollo-server/) – A production-ready JavaScript GraphQL server that connects to any microservice, API, or database. Compatible with all popular JavaScript frameworks and deployable in serverless environments.
```
Comment (jerelmiller) [inline README.md:47]: Since the apollo client links were updated to the GitHub repos, should this one be updated in the same way?

**Review by jerelmiller — APPROVED** [review]
Looks good to me. Was curious about another potential link update, but nothing blocking. Feel free to merge when you're ready 🙂

File: README.md (hunk 1 above)
Comment (Meschreiber) [reply README.md:47]: Good call on consistency. Previously the mobile clients liked to GH, so I updated ACW to do the same. Happy to do the same for Server.

Comment (Meschreiber) [inline README.md:47]:
```suggestion
- [Apollo Server](https://github.com/apollographql/apollo-server) – A production-ready JavaScript GraphQL server that connects to any microservice, API, or database. Compatible with all popular JavaScript frameworks and deployable in serverless environments.
```

**Pull request merged**